include/remill/BC/Compat/CTypes.h

```diff
@@ -1,30 +0,0 @@
-/*
- * Copyright (c) 2017 Trail of Bits, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-#pragma once
-
-#if __has_include(<llvm-c/Types.h>)
-#  include <llvm-c/Types.h>
-
-namespace llvm {
-
-// TODO(pag): This is a rather ugly hack; had some issues with anvill not
```

Comment:
I don't see this file included anywhere in Anvill so I'm assuming this doesn't apply anymore.